docs/getting-started.md

```diff
@@ -0,0 +1,170 @@
+# Getting started
+
+These quickstarts run on <span>WildFly 10</span>.
+
+Prior to running the quickstarts you should read this entire document and have completed the following steps:
+
+* [Start and configure the Red Hat SSO Server](#keycloak)
+* [Start and configure the <span>WildFly</span> Server](#wildfly)
+
+Afterwards you should read the README file for the quickstart you would like to deploy. See [examples](#examples) for
+a list of the available quickstarts.
+
+If you run into any problems please refer to the [troubleshooting](#troubleshooting) section.
+
+
+## Use of <span>KEYCLOAK_HOME</span> and <span>WILDFLY_HOME</span> Variables
+
+The quickstart README files use the replaceable value <span>KEYCLOAK_HOME</span> to denote the path to the Red Hat SSO installation and the
+value <span>WILDFLY_HOME</span> to denote the path to the <span>WildFly</span> installation. When you encounter this value in a README file, be sure
+to replace it with the actual path to your installations.
+
+
+## System Requirements
+
+The applications these projects produce are designed to be run on <span>WildFly</span> Application Server 10.
+
+All you need to build these projects is Java 8.0 (Java SDK 1.8) or later and Maven 3.1.1 or later.
+
+
+## <a id="keycloak"></a>Start the Red Hat SSO Server
```

Comment:
I believe we don't want to mention "Red Hat SSO Server" in  this chapter. :)

Reply:
Thanks, updated.